services/explorer/src/pages/AddressInfo/AddressInfo.effects.tsx

```diff
@@ -1,35 +1,68 @@
-import { useCallback, useEffect, useState } from 'react';
+import cleanDeep from 'clean-deep';
+import { useEffect, useState } from 'react';
+import { useQuery } from 'urql';
+import { ethers } from 'ethers';
+import { accountQuery } from '../../api/graphql/queries/account';
+import { useExplorerNetwork } from '../../hooks';
 import {
-  getMockedAddresInfoDetails,
   getMockedDagBlocks,
   getMockedTransactions,
   getMockPbftBlocks,
 } from '../../api/mocks';
 import { useExplorerLoader } from '../../hooks/useLoader';
 import { AddressInfoDetails, BlockData, Transaction } from '../../models';
 
-export const useAddressInfoEffects = (txHash: string) => {
+export const useAddressInfoEffects = (account: string) => {
+  const { currentNetwork } = useExplorerNetwork();
   const [transactions, setTransactions] = useState<Transaction[]>();
   const [dagBlocks, setDagBlocks] = useState<BlockData[]>();
   const [pbftBlocks, setPbftBlocks] = useState<BlockData[]>();
   const [addressInfoDetails, setAddressInfoDetails] =
     useState<AddressInfoDetails>();
+  const [{ fetching, data }] = useQuery({
+    query: accountQuery,
+    variables: cleanDeep({
```

Comment:
Not sure if you need cleanDeep here. Because account is always needed, right? And the hook won't work if account is missing (because of pause). So I don't think you need cleanDeep here